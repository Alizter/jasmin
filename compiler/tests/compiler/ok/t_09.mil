// reuse the same number for different names, renumber before
//   type checking

extern fn test(x.1, y.1 : stack u64) -> stack u64 {
  x.1 += y.1;
  return x.1;
}

/*
START:CMD
ARG="typecheck,print[renum][fun=test]"
END:CMD
*/
