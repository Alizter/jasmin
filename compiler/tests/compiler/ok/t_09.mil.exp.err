Parsing tests/compiler/ok/t_09.mil as MIL

Processed file tests/compiler/ok/t_09.mil
