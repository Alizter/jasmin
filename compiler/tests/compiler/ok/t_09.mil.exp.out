type checking module in all function(s)
   REMOVED_TIME
>> renum:
extern fn test(x.1 : stack u64, y.1 : stack u64) -> stack u64 {
  x.1 += y.1;
  return x.1;
}


total transformation time: REMOVED_TIME

total time (with save/print): REMOVED_TIME
